.github/workflows/gradle-check.yml

```diff
@@ -31,8 +27,17 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
 
+      - name: Get changed files
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v44
+        with:
+          files_ignore: |
+            release-notes/**
+            .github/**
```

Comment:
We cannot skip files under `.github`, if you wanted to modify this to specific files - code owners, this is OK with me.

Reply:
Are files under .github related to gradle check in anyway?

Reply:
They never are, these are github workflows. And the workflow itself runs on pull request target, so it doesn't impact what's used on the PR itself with the changes to it.